Module_07/README.md

```
@@ -0,0 +1,144 @@
+public:
+    Pair(T a, T b) : first(a), second(b) {}
+};
+```
+
+## SFINAE (Substitution Failure Is Not An Error)
+
+`SFINAE` is a rule in C++ that allows the compiler to silently ignore a template when its instantiation is invalid, instead of throwing a compilation error.
+
+___
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)